site/app/templates/course/CourseMaterials.twig

```diff
@@ -297,10 +269,40 @@
     function setNewDateTime(me, path) {
         // pass filename to server to record the new date and time of the file to be released
         var newDateTime = $(me).val();
-
         changeNewDateTime(path, newDateTime);
+        var now = new Date();
+
+        function pad(str){
+            return ('0'+str).slice(-2);
+        }
+
+        var date = now.getFullYear()+'-'+pad(now.getMonth()+1)+'-'+pad(now.getDate());
+
+        var time = pad(now.getHours())+":"+pad(now.getMinutes())+":"+pad(now.getSeconds());
+        var currentDT = date+' '+time;
+        var neverDT = (now.getFullYear()+10)+'-'+pad(now.getMonth()+1)+'-'+pad(now.getDate())+' '+time;
+
+        //get the value in each file so the color can be assigned
+        //based on the time chosen
+        var fileDT = newDateTime;
+
+        //also custom colors for this page for readability
+        if(new Date(fileDT).getTime()<=new Date(currentDT).getTime()){
+            me.style.backgroundColor = "#87db88";
+
+            return "#87db88";
+        }
+        else if(fileDT>=neverDT){
```

Comment:
I think you need to change fileDT and neverDT to Js Date Objects before comparison

Reply:
ah I missed thst guy somehow